web/themes/custom/pineapple/sass/components/paragraphs/_accordion.scss

```diff
@@ -0,0 +1,48 @@
+.accordion-title {
+  background-color: $light-gray;
+  color: $primary;
+  width: 100%;
+  text-align: left;
+  padding: $unit *2;
+  transition: 0.4s;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  &:hover,
+  &:focus {
+    color: $white;
+    background-color: $gray;
```

Comment:
Can you please check if this colour combination is accessible?

Reply:
wasn't good enough. Fixed now.